tests/pkg_aversiveplusplus/Makefile

```diff
@@ -1,6 +1,11 @@
 APPLICATION = pkg_aversiveplusplus
 include ../Makefile.tests_common
 
+# temporarily disable for CI
+ifeq ($(RIOT_CI_BUILD), 1)
```

Comment:
Try ```make buildtest```. The whitelisting causes "make info-boards-supported" to be empty. I assumed that would skip it for the CI.